src/autopas/containers/verletClusterLists/VerletClusterLists.h

```diff
@@ -200,6 +198,7 @@ class VerletClusterLists : public ParticleContainerInterface<Particle>, public i
         // don't simply copy haloParticle over iter. This would trigger a dataRace with other regionIterators that
         // overlap with this region.
         it->setR(haloPos);
+        it->setRAtRebuild();
```

Comment:
is this correct? If I remember correctly, this function is like moving the halo particle so why assume we rebuild it's neighbor list?

Btw: Do we even build neighbor lists for halo particles? I think in the 3 Body case we do but I'm not sure for the 2 Body case.

Reply:
Corrected this - setting the rebuild position as the value stored in halo particle being added